docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,883 @@
+   | CYCL_HRS                       | ("HH1" "HH2")     | "00"                                                   |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+   | EXTRN_MDL_NAME_ICS             |  "FV3GFS"         | "FV3GFS"                                               |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+   | EXTRN_MDL_NAME_LBCS            |  "FV3GFS"         | "FV3GFS"                                               |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+   | FV3GFS_FILE_FMT_ICS            |  "nemsio"         | "grib2"                                                |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+   | FV3GFS_FILE_FMT_LBCS           |  "nemsio"         | "grib2"                                                |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+   | WTIME_RUN_FCST                 |  "04:30:00"       | "01:00:00"                                             |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+   | USE_USER_STAGED_EXTRN_FILES    |  "FALSE"          | "TRUE"                                                 |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+   | EXTRN_MDL_SOURCE_BASE_DIR_ICS  |  ""               | "/scratch2/BMC/det/UFS_SRW_app/v1p0/model_data/FV3GFS" |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+   | EXTRN_MDL_FILES_ICS            |  ""               | "gfs.pgrb2.0p25.f000"                                  |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+   | EXTRN_MDL_SOURCE_BASEDIR_LBCS  |  ""               | "/scratch2/BMC/det/UFS_SRW_app/v1p0/model_data/FV3GFS" |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+   | EXTRN_MDL_FILES_LBCS           |  ""               | "gfs.pgrb2.0p25.f006"                                  |
+   +--------------------------------+-------------------+--------------------------------------------------------+
+
+ 
+To get started, make a copy of ``config.community.sh``. From the ``ufs-srweather-app`` directory, run:
+
+.. code-block:: console
+
+   cd regional_workflow/ush
+   cp config.community.sh config.sh
+
+The default settings in this file include a predefined 25-km :term:`CONUS` grid (RRFS_CONUS_25km), the :term:`GFS` v15.2 physics suite (FV3_GFS_v15p2 CCPP), and :term:`FV3`-based GFS raw external model data for initialization.
+
```

Comment:
Has CCPP been defined previously?

Reply:
It's defined in the Introduction, but I've added glossary links now, too. Don't want to link people to death, but I think it's good to add in this case. Let me know if you want links to other terms added anywhere (or more terms added to the glossary). I tried not to link multiple times in one paragraph, but other than that, I think term links are a good addition in most places.

Reply:
Sounds good, thanks!  I'll take another look for other terms.